hack/components/bump-linux-bridge.sh

```diff
@@ -22,8 +22,6 @@ echo 'Get linux-bridge tag'
 LINUX_BRIDGE_TAG=$(git-utils::get_component_tag ${LINUX_BRIDGE_PATH})
 
 echo 'Build container image with linux-bridge binaries'
-LINUX_BRIDGE_TAR_CONTAINER_DIR=/usr/src/github.com/containernetworking/plugins/bin
```

Comment:
we can (and should) use it [here](https://github.com/kubevirt/cluster-network-addons-operator/pull/947/files#diff-5fba3b663166801728429b904315b89d6f1b898012f6f879dfd9178f5610a824L45-L46)